app/models/group.rb

```diff
@@ -0,0 +1,11 @@
+class Group < ApplicationRecord
+  # relationships
+  belongs_to :user
+  has_and_belongs_to_many :entities
+  # validations
+  validates :name, presence: true, length: { minimum: 3, maximum: 50 }
+
```

Comment:
- [ ] Please ensure that you add validation for the user input amount for the Group model to make sure that anyone with bad intentions cannot compromise your app.

Reply:
ok, thanks